PinpointKit/PinpointKit/Sources/Core/PinpointKit.swift

```diff
@@ -86,7 +86,18 @@ extension PinpointKit: SenderDelegate {
     public func sender(_ sender: Sender, didFailToSend feedback: Feedback?, error: Error) {
         if case MailSender.Error.mailCanceled = error { return }
         
-        NSLog("An error occurred sending mail: \(error)")
+        let alert = UIAlertController(
```

Comment:
Can you move this UIAlertController and presentation to the default protocol extension on ` func pinpointKit(_ pinpointKit: PinpointKit, didFailToSend feedback: Feedback, error: Error) {}`?